File: fbnq-research/fbnq.c

#include <stdio.h>
#include <time.h>

int fibonacci(int n) {
    if (n <= 1) return n;
    return fibonacci(n-1) + fibonacci(n-2);
}

int main() {
    clock_t start = clock();
    int result = fibonacci(37);
    double elapsed = (double)(clock() - start) / CLOCKS_PER_SEC * 1000;
    
    printf("斐波那契第37项: %d\n", result);
    printf("计算耗时: %.3f 毫秒\n", elapsed);
    return 0;
}

File: fbnq-research/fbnq.go

package main

import (
	"fmt"
	"time"
)

func fibonacci(n int) int {
	if n <= 1 {
		return n
	}
	return fibonacci(n-1) + fibonacci(n-2)
}

func main() {
	start := time.Now()
	result := fibonacci(37)
	elapsed := time.Since(start)
	
	fmt.Printf("斐波那契第37项: %d\n", result)
	fmt.Printf("计算耗时: %v\n", elapsed)
}
